src/DynamoCoreWpf/Views/HomePage/HomePage.xaml.cs

```diff
@@ -0,0 +1,572 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Windows;
+using System.Windows.Controls;
+using Dynamo.Models;
+using Dynamo.UI.Controls;
+using Dynamo.Utilities;
+using Dynamo.Wpf.UI.GuidedTour;
+using DynamoUtilities;
+using Microsoft.Web.WebView2.Core;
+using Microsoft.Web.WebView2.Wpf;
+
+
+namespace Dynamo.UI.Views
+{
+    /// <summary>
+    /// Interaction logic for HomePage.xaml
+    /// </summary>
+    public partial class HomePage : UserControl, IDisposable
+    {
+        // These are hardcoded string and should only change when npm package structure changed or image path changed
+        private static readonly string htmlEmbeddedFile = "Dynamo.Wpf.Packages.HomePage.build.index.html";
+        private static readonly string jsEmbeddedFile = "Dynamo.Wpf.Packages.HomePage.build.bundle.js";
+        private static readonly string fontStylePath = "Dynamo.Wpf.Views.GuidedTour.HtmlPages.Resources.ArtifaktElement-Regular.woff";
+        private static readonly string virtualFolderName = "embeddedFonts";
+        private static readonly string virtualFolderPath = Path.Combine(Path.GetTempPath(), virtualFolderName);
+
+        private string fontFilePath;
+
+        private StartPageViewModel startPage;
+
+        /// <summary>
+        /// The WebView2 Browser instance used to display splash screen
+        /// </summary>
+        internal WebView2 webView;
```

Comment:
please use the DynamoWebView2 class

Reply:
Thank you, @pinzart90 ! Will replace with the Dynamo versions now!

Reply:
I made the swap, and it all works great! Thanks again @pinzart90